diff --git a/node_modules/remark-stringify/lib/escape.js b/node_modules/remark-stringify/lib/escape.js
index 3343d82..c2f933b 100644
--- a/node_modules/remark-stringify/lib/escape.js
+++ b/node_modules/remark-stringify/lib/escape.js
@@ -5,6 +5,7 @@ var alphanumeric = require('is-alphanumeric')
 var whitespace = require('is-whitespace-character')
 var escapes = require('markdown-escapes')
 var prefix = require('./util/entity-prefix-length')
+var identity = require('./util/identity')
 
 module.exports = factory
 
@@ -48,7 +49,7 @@ var blankExpression = /\n\s*$/
 
 // Factory to escape characters.
 function factory(options) {
-  return escape
+  return options.escape ? escape : identity
 
   // Escape punctuation characters in a node’s value.
   function escape(value, node, parent) {
diff --git a/node_modules/remark-stringify/lib/set-options.js b/node_modules/remark-stringify/lib/set-options.js
index 1f5c343..22a3405 100644
--- a/node_modules/remark-stringify/lib/set-options.js
+++ b/node_modules/remark-stringify/lib/set-options.js
@@ -16,7 +16,8 @@ var maps = {
   listItemIndent: {tab: true, mixed: true, 1: true},
   emphasis: {_: true, '*': true},
   strong: {_: true, '*': true},
-  fence: {'`': true, '~': true}
+  fence: {'`': true, '~': true},
+  escape: {true: true, false: false},
 }
 
 // Expose `validate`.
